### ISSUE TITLE
loss explosion

### ISSUE THREAD

**Issue by RoderickGu** [huggingface/pytorch-transformers#1138]
## ❓ Questions & Help

I am using Bert-CRF model to do Named Entity Recognition task. I am using the average of the last four layers as the input of CRF model.  But the loss will increase and become Nan in a few batches. Anyone meet that problem before? Any suggestions will be appreciated!

**Comment by RoderickGu** [comment]
just because of a large learning rate
